controllers/datadogagent/feature/orchestratorexplorer/feature.go

```diff
@@ -74,9 +78,19 @@ func (f *orchestratorExplorerFeature) Configure(dda *v2alpha1.DatadogAgent) (req
 
 	if orchestratorExplorer != nil && apiutils.BoolValue(orchestratorExplorer.Enabled) {
 		reqComp.ClusterAgent.IsRequired = apiutils.NewBoolPointer(true)
+		reqContainers := []apicommonv1.AgentContainerName{apicommonv1.CoreAgentContainerName}
+
+		// Process Agent is not required as of agent version 7.51.0
+		if nodeAgent, ok := dda.Spec.Override[v2alpha1.NodeAgentComponentName]; ok {
+			if f.processAgentRequired = nodeAgent.Image != nil &&
+				!utils.IsAboveMinVersion(component.GetAgentVersionFromImage(*nodeAgent.Image), "7.51.0"); f.processAgentRequired {
```

Comment:
sorry i wasn't clear in my comment earlier - here's my suggested edit:

```suggestion
		if nodeAgent, ok := dda.Spec.Override[v2alpha1.NodeAgentComponentName]; ok {
			if nodeAgent.Image != nil && !utils.IsAboveMinVersion(component.GetAgentVersionFromImage(*nodeAgent.Image), "7.51.0") {
			f.processAgentRequired = true
```

Reply:
Done